examples/htool.c

```diff
@@ -536,6 +536,36 @@ static int command_passthrough_enable(const struct htool_invocation* inv) {
       /*version=*/0, NULL, 0, NULL, 0, NULL);
 }
 
+static int command_srtm(const struct htool_invocation* inv) {
+  struct libhoth_device* dev = htool_libhoth_device();
+  if (!dev) {
+    return -1;
+  }
+
+  const char* measurement;
+  if (htool_get_param_string(inv, "measurement", &measurement) != 0) {
+    return -1;
+  }
+
+  size_t measurement_length = strlen(measurement);
```

Comment:
I don't think it will work to pass raw binary data here. (a) it makes to command pretty hard to use and (b) if the data contains any null bytes, the strlen check will terminate early.

I think there are a couple ways to do this that might make sense:

1. Pass a file with binary data that htool can read
2. Pass a hex string which we decode to binary

1 seems a little easier to implement, but 2 seems easier to use.